pkg/api/handlers/libpod/artifacts.go

```diff
@@ -162,6 +162,50 @@ func RemoveArtifact(w http.ResponseWriter, r *http.Request) {
 	utils.WriteResponse(w, http.StatusOK, artifacts)
 }
 
+func BatchRemoveArtifact(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm not sure if having two endpoints for artifact removal is a good idea.

Reply:
Other resources do this by calling list and then the delete endpoint for each listed item. That means a lot more API roundtrips. I am fine with a extra endpoint to handle multiple removals.